content/en/docs/install-setup/installing/boot/ingress.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Ingress
+linktitle: Ingress
+description: Configure ingress
+weight: 65
+---
+
+## Ingress
+
+If you don't specify anything in your [jx-requirements.yml](https://github.com/jenkins-x/jenkins-x-boot-config/blob/master/jx-requirements.yml) file then boot will default to using HTTP (rather than HTTPS) and using [nip.io](https://nip.io/) as the DNS mechanism.
```

Comment:
```suggestion
If you don't specify anything in your [jx-requirements.yml](https://github.com/jenkins-x/jenkins-x-boot-config/blob/master/jx-requirements.yml) file then Boot will default to using HTTP (rather than HTTPS) and using [nip.io](https://nip.io/) as the DNS mechanism.
```